### PR TITLE
TableReader add

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Just to add the possibility to read the all the lines contents in input file.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

**Comment by kerautret** [comment]
@dcoeurjo or @JacquesOlivierLachaud can we merge ? (small PR ;) but i use in other remote) ;)

File: src/DGtal/io/readers/TableReader.h [hunk 1]
```diff
@@ -113,7 +113,31 @@ namespace DGtal
     getColumnElementsFromInputStream (std::istream &in,
                                       unsigned int aPosition);
 
+    /**
+     * Method to import a vector where each element contains the line
+     * elements of a given file. Blank line or line beginning with
```
Comment (dcoeurjo) [inline src/DGtal/io/readers/TableReader.h]: how "elements" are defined ? are they "space", "comma", "tab",.. separated values ?

Comment (kerautret) [inline src/DGtal/io/readers/TableReader.h]: space or tab I add it (and I add tabs in the test file pointList1.pl

Comment (dcoeurjo) [inline src/DGtal/io/readers/TableReader.h]: is it documented somewhere ?

Comment (dcoeurjo) [inline src/DGtal/io/readers/TableReader.h]: don't remember the original TableReader PR  :/

Comment (kerautret) [inline src/DGtal/io/readers/TableReader.h]: no but I add it ;)

**Comment by dcoeurjo** [comment]
;)
merging

**Pull request merged**